transformations/tests/test_single_column_coalesced.py

```diff
@@ -796,6 +804,9 @@ def test_single_column_coalesced_hoist_openacc(frontend, horizontal, vertical, b
     analysis = HoistTemporaryArraysAnalysis(dim_vars=(vertical.size,))
     synthesis = SCCHoistTemporaryArraysTransformation(block_dim=blocking)
 
+    # Check that blocking size has not been redefined
+    assert driver.symbol_map[blocking.size].type.module
```

Comment:
Is this missing some kind of comparison maybe? E.g., `driver.symbol_map[blocking.size].type.module.lower() == 'block_dim_mod'`?